.github/workflows/test.yml

```diff
@@ -136,13 +136,14 @@ jobs:
         shell: powershell
         run: |
             Set-PSRepository PSGallery -InstallationPolicy Trusted
-            Install-Module SqlServer -ErrorAction Stop
+            Install-Module SqlServer
 
       - name: Install SQL Server 2019
         shell: powershell
         run: |
-          choco feature disable -name=exitOnRebootDetected
-          choco install sql-server-2019
+          choco feature disable --name="'exitOnRebootDetected'"
+          $ErrorActionPreference = 'SilentlyContinue'
```

Comment:
I don't understand the powershell code, so it's hard for me to review. Could there be bad implications to "SilentlyContinue"?

Reply:
Yes. The installation could fail miserably and cause errors later in the pipeline.

The right thing to do here would be to restart the computer...